README.md

```diff
@@ -1,12 +1,24 @@
 # xblock-group-project-v2
 
-
 This XBLock is experimental reimplementation of [Group Project XBlock](https://github.com/edx-solutions/xblock-group-project).
 
 It does *not* work properly in LMS yet.
 
 ## Features
```

Comment:
Would moving this to top be better: https://github.com/open-craft/xblock-group-project-v2/pull/103/files/be887f0fd72181dfd173b0b79820be6722e6c3dc#diff-04c6e90faac2675aa89e2176d2eec7d8R64